modules/cross-account-member/main.tf

```
@@ -14,8 +10,9 @@ module "account" {
 # Send invite from administrator account
 module "member" {
   source = "../member"
+
   providers = {
-    aws = aws.administrator
+    aws = aws.admin
```

Changing this value has impacts on existing implementations. Let's leave it as `administrator` to minimize the impact to the user.

```suggestion
    aws = aws.administrator
```